docs/reference/api/rtos/rtos.md

```diff
@@ -1,14 +1,18 @@
 <h2 id="rtos-api">RTOS overview</h2>
 
-The Arm Mbed RTOS is a C++ wrapper over the Keil RTX code. For more information about Keil RTX, check [the Keil CMSIS-RTOS tutorial](https://github.com/ARM-software/CMSIS/raw/master/CMSIS/Documentation/RTX/CMSIS_RTOS_Tutorial.pdf) and [the element14 introduction to Keil RTX](https://www.element14.com/community/docs/DOC-46650/l/arm-keil-rtx-real-time-operating-system-overview). You can use these resources as a general introduction to RTOS principles; it is important to be familiar with the concepts behind an RTOS in order to understand this guide.
+Mbed-OS RTOS provide C++ APIs to manage objects like thread, synchronization objects and timer. It also provides interfaces for attaching application
```

Comment:
Minor nit:
Change to "The Mbed OS RTOS provides C++ APIs to manage objects like threads, synchronization objects, and timers."

Reply:
@kegilbert - Change done, please review.